packages/core/src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -238,6 +242,7 @@ export default class PaypalCommercePaymentStrategy implements PaymentStrategy {
         const returnObject = {
             'client-id': clientId,
             'merchant-id': merchantId,
+            'buyer-country': 'US',
```

Comment:
Should we pass the buyer country as 'US' for users from UK?

Reply:
It was add for test to avoid using VPN